spec/factories/content_type.rb

```diff
@@ -1,37 +1,9 @@
 FactoryGirl.define do
-  factory :validatable_content_type, class: ContentType do
-    sequence(:class_name) {|n| "ContentItem#{n}" }
-    sequence(:controller) {|n| "content_item_#{n}" }
-    sequence(:humanized_plural) {|n| "Content Item #{n}s" }
-    sequence(:humanized) {|n| "Content Item #{n}" }
-
-    factory :saveable_content_type  do
-      sequence(:description) {|n| "Content item #{n} content type" }
-    end
+  factory :content_type do
+    sequence(:class_name) { |n| "ContentItem#{n}" }
+    sequence(:controller) { |n| "content_item_#{n}" }
+    sequence(:humanized_plural) { |n| "Content Item #{n}s" }
+    sequence(:humanized) { |n| "Content Item #{n}" }
+    sequence(:description) { |n| "Content item #{n} content type" }
   end
-
-
- singleton_content_types_needed_to_create_models = [
```

Comment:
I removed the few places these factories were used in the model specs so they definitely are not needed anywhere. I think we can add traits to the `:content_type` factory to implement these quite cleanly if/when we need them in future.